pages/portfolio.html

```diff
@@ -4,9 +4,52 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>GELLYCODES::portfolio</title>
+  <link href="/styles/portfolio.css" rel="stylesheet">
 </head>
 <body>
-  
+  <header>
+  </header>
+  <nav>
+    <ul>
+      <li><a href="/pages/about.html">about</a></li>
+      <li>
+        <a href="/pages/index.html"><h1>gellycodes</h1></a>
+      </li>
+      <li><a href="/pages/portfolio.html">portfolio</a></li>
+  </ul>
+  </nav>
+  <main>
+    <a href="https://github.com/wet-bulb/task-list-api">
+      <div>
+        <img src="/assets/tl.png" alt="">
+        <h2>Task List</h2>
+        <p>Slack-integrated, RESTful to-do list API</p>
+      </div>
+    </a>
```

Comment:
These are a list of projects, so placing them in a ul as list items would be a good semantic fit. And consider adding a class to help indicate that each chunk of html represents a project. This can help our eyes stay oriented in our code.